### ISSUE TITLE
RAMP submission explanation

### ISSUE THREAD

**Issue by lucyleeow** [ramp-kits/air_passengers#11]
The notebook just points to: https://paris-saclay-cds.github.io/ramp-docs/ramp-workflow/stable/using_kits.html but does not explain the basic details:

- requires `estimator.py` file
- define a `get_estimator` fun with the file which needs to return a sklearn pipeline or estimator

**Comment by tomMoral** [comment]
Yes explaining this after the first submission would be very helpful! :)
